src/cloud.c

```diff
@@ -63,19 +64,43 @@ static bool cloud_receive_message(const char *exchange,
 				  const char *routing_key,
 				  const char *body, void *user_data)
 {
-	/* TODO: Parser body message */
+	json_object *jso;
+	json_object *jobjkey;
+	struct l_queue *list;
+	const char *id;
+
+	jso = json_tokener_parse(body);
+	if (!jso)
+		return false;
+
+	if (!json_object_object_get_ex(jso, "id", &jobjkey))
```

Comment:
Magic string `"id"`

Reply:
Theses strings are from KNoT AMQP API. For now, we decide to stay with "magic strings" and after create a better way to manage the API on knotd.